app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt

```diff
@@ -66,13 +63,13 @@ class EditActivity : AppCompatActivity() {
                 ExifInterface.TAG_GPS_TIMESTAMP,
                 ExifInterface.TAG_IMAGE_LENGTH,
                 ExifInterface.TAG_IMAGE_WIDTH,
-                ExifInterface.TAG_ISO,
```

Comment:
For reviewers:
As per linked issue I migrated to `TAG_ISO` to `TAG_ISO_SPEED_RATINGS` but this ISO_SPEED_Ratings is deprecated in `androidx.exifinterface.media.ExifInterface` so I migrated it to `TAG_PHOTOGRAPHIC_SENSITIVITY`

Ref : https://developer.android.com/reference/kotlin/androidx/exifinterface/media/ExifInterface#TAG_ISO_SPEED_RATINGS()